### PR TITLE
plist for isync brew services

### PR DESCRIPTION
**Pull request by cacology**
- [Yes ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [Yes] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [Yes] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [Yes] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

Added a plist to isync so that "brew services start isync" would start syncing mail based on the "Periodic" group.

-----

Respectfully submitted,
-James

### CODE REVIEW

File: Formula/isync.rb [hunk 1]
```diff
@@ -37,6 +37,45 @@ def install
     system "make", "install"
   end
 
+  def caveats; <<-EOS.undent
+    If you setup your ~/.mbsync file with a Group called "Periodic", 'brew services start isync' will sync that group every five minutes.  Typically, those rules would check your inbox, trash, and sent mail.
```
Comment (dunn) [inline Formula/isync.rb]: Is this Homebrew-specific information? If not, let's not include it.

Comment (cacology) [inline Formula/isync.rb]: Seems fine to me.  An isync user could figure out the "Periodic" group was what they wanted based on the plist.  If they can't figure that out, then the comment probably won't be much help.

Profoundly ignorant question: do I make another pull request with that change?  Or can you merely alter it?  (if the later, go ahead)

Comment (dunn) [inline Formula/isync.rb]: You can add another commit to this branch, or amend the first commit and force-push.

**Comment by dunn** [comment]
Thanks! Could you squash to one commit and add `revision 1` below the `sha256` line so that everyone gets the update?

**Comment by dunn** [comment]
Thanks! Merged as 4c182aaca49dc83dfa34111371bac3d5e8cf6da8.

**Comment by cacology** [comment]
Thank you too and thanks for your welcome guidance above--it's exciting to learn to contribute and you're generous to explain.

**Pull request closed without merging**